channels/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+var numbers = []string{"1", "2", "3", "4", "5", "6", "7", "8", "9"}
+
+func main() {
+	strings := []string{"1211Nikhil34Shar2ma", "H33a4r6shi777tS7687harma", "S45346eem568568aSh86a9r8ma"}
+	c := make(chan string)
+	correctStrings := getCorrectStrings(strings, c)
+	fmt.Println(correctStrings)
+}
+
+func getCorrectStrings(inputStrings []string, c chan string) []string {
+	var filteredString string
```

Comment:
There is no need for `filteredString` variable declaration as this is scoped only inside the for loop.